### ISSUE TITLE
Add functionality for ChAs to be able to use new registration

### ISSUE THREAD

**Issue by shaun-technovation** [Iridescent-CM/technovation-app#4080]
This is the initial work for getting chapter ambassadors using the new registration flow.

Create new form fields and back-end functionality in order for chapter ambassadors to be able to use the new registration flow.

With our existing pattern (under the directory`new_registration/components/`), we have:
- `JudgeStepTwo.vue`
- `MentorStepTwo.vue`
- `StudentStepTwo.vue`

Following this pattern, a new step for chapter ambassadors should be created.

Here are the current fields for chapter ambassadors:
<img src="https://github.com/Iridescent-CM/technovation-app/assets/58957909/0fc258bf-d7d1-448a-bc3f-643635642329" width="600">

And then in `app/controllers/new_registration_controller.rb` we need to add functionality to save the chapter ambassador to the database. The current save functionality is in ` app/controllers/chapter_ambassador/signups_controller.rb`.

Notes:
In order to develop this (and access the new step for chapter ambassadors), a new profile on step one would have to be created (I think), but this new profile shouldn't be committed; a chapter ambassador profile type should not be available as an option, chapter ambassadors will need an invite token (future work) to be able use the new registration flow.

**Comment by shaun-technovation** [comment]
This one is on QA, ready for an initial review/feedback. One thing, which we'll see on QA, is that we need a new avatar/icon for the chapter ambassador profile type.

**Comment by dboyer** [comment]
I submitted a request for a new avatar for ChAs.

On QA, I tested:

- Sent an invite to a new email address
- Received the invite and the link took me to the new registration page where the ChA profile was selected
- I was able to fill out the information to create an account. I set my location and then had a message on my dashboard that my profile was pending approval.
- On the admin page, I went to their profile and set their status to approved.
- I logged back into my ChA account and because I set my location to the US had a message to get a background check
- I repeated the process for a non-US ChA and was able to get access to my dashboard and start adding information and viewing participants
